core/components/organisms/select/Select.tsx

```diff
@@ -252,22 +262,19 @@ export const Select = React.forwardRef<SelectMethods, SelectProps>((props, ref)
 
   return (
     <SelectContext.Provider value={contextProp}>
-      <div
-        data-test="DesignSystem-Select"
-        aria-haspopup="listbox"
-        aria-expanded={openPopover}
-        style={triggerStyle}
-        {...baseProps}
-      >
+      <div data-test="DesignSystem-Select" aria-haspopup="listbox" aria-expanded={openPopover} {...baseProps}>
         <Popover
           open={openPopover}
           onToggle={onToggleHandler}
           className="mt-3"
           triggerClass="d-block"
+          position="bottom-start"
           customStyle={popoverStyle}
           boundaryElement={boundaryElement}
           appendToBody={appendToBody}
-          trigger={<SelectTrigger aria-controls="select-listbox" {...triggerOptions}></SelectTrigger>}
+          trigger={
+            trigger ? trigger : <SelectTrigger aria-controls="select-listbox" {...triggerOptions}></SelectTrigger>
+          }
```

Comment:
Please create a separate variable for `trigger ? trigger : <SelectTrigger aria-controls="select-listbox" {...triggerOptions}></SelectTrigger>` and then pass that variable as a prop

Reply:
done